AUTHORS

```diff
@@ -22,3 +22,4 @@ Ruslan Kovalov <ruslan.kovalyov@gmail.com>
 Venafi, Inc.
 Vladimir Rutsky <vladimir@rutsky.org>
 Ximin Luo <infinity0@gmx.com>
+Laël Cellier
```

Comment:
I’m unwilling to provide a valid e‑mail address that can receive span. However, I’d be glad to share my irc or xmpp address.
Sorry for the new line. I’ll add it